unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/PrivateChatWindow/PrivateChatWindowController.cs

```diff
@@ -305,7 +305,7 @@ private void HandleInputFieldSelected()
         deactivatePreviewCancellationToken = new CancellationTokenSource();
         DeactivatePreview();
         // The messages from 'conversationUserId' are marked as read if the player clicks on the input field of the private chat
-        MarkUserChatMessagesAsRead();
+        //MarkUserChatMessagesAsRead();
```

Comment:
why this got removed?

Reply:
As I was seeing, it is not needed since we are already sending the `MarkAsRead` in these 2 cases:
- When the chat window is opened.
- When a new message comes into the chat and it is already opened.

There shouldn't be more cases to take into account to keep kernel informed about the reads. To send another `MarkAsRead` each time the user puts the focus in the input textbox is redundant.

Anyway, for now I left the line commented in case we see any strange behavior in the integration stage 👍